package.json

```diff
@@ -4,64 +4,55 @@
   "version": "1.0.0",
   "license": "Apache-2.0",
   "scripts": {
-    "start": "if-env NODE_ENV=production && npm run -s serve || npm run -s dev",
-    "build": "preact build --no-prerender",
-    "serve": "preact build --no-prerender && preact serve",
+    "build": "preact build",
```

Comment:
As you mentioned, this works for me:
```suggestion
    "build": "preact build --no-prerender",
```
Without it, I see this error when running `npm run build`:
```
ReferenceError: window is not defined
method: render
at: /Users/dj/Code/architecture-viewer-swaraj/src/components/app.js:197:16

Source code:

	}
	render() {
		let repoUrl = window.location.href.replace("pages/", "");
		if (this.state.graphData && this.state.stepData) {
			return (
				<div id="container">

This is most likely caused by using DOM or Web APIs.
Pre-render runs in node and has no access to globals available in browsers.

Consider wrapping code producing error in: 'if (typeof window !== "undefined") { ... }'

Alternatively use 'preact build --no-prerender' to disable prerendering.

See https://github.com/developit/preact-cli#pre-rendering for further information.npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! architecture-viewer@1.0.0 build: `preact build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the architecture-viewer@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/dj/.npm/_logs/2020-08-06T02_05_14_313Z-debug.log
```